spec/game.test.js

```diff
@@ -0,0 +1,107 @@
+const { describe } = require('jest-circus');
+const Game = require('../src/game');
+
+beforeEach(() => {
+  game = new Game();
+});
+
+describe('Game', () => {
+  test('has an empty array of frames', () => {
+    expect(game.frames).toEqual([]);
```

Comment:
I don't think tests like these are overly helpful - if you wanted to change game.frames to be a frame object instead of an array, even if the program still works the way you want it to (ie the behaviour is the same) the test will now fail.